com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Math/Round/StepNode.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class StepNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "Step", // Name
+            "Out = step(Edge, In);",
+            new ParameterDescriptor("In", TYPE.Vector, Usage.In),
+            new ParameterDescriptor("Edge", TYPE.Vector, Usage.In, new float[] { 1f, 1f, 1f, 1f }),
```

Comment:
Do we want to put the `Edge` above` In` as the old node?

Reply:
I switched this one on purpose because in every other node with more than one port, the data input is first and putting it first is one of the principles in our style guide.  I realzie that it's second because of the order it goes into the intrinsic function, but I think it should be consistent with the other nodes.